processor/htmlproc/htmlproc.go

```diff
@@ -45,6 +47,9 @@ type Config struct {
 // htmldoc.HTMLResponse then runs the specified htmltask.HTMLTasks one by one.
 // The Processor fails immediately when some HTMLTask encounters an error.
 func NewHTMLProcessor(config Config) processor.Processor {
+	if len(config.TaskSet) == 0 {
```

Comment:
`config.TaskSet == nil`, if we indeed want to provide a way to make the set empty. Otherwise "nil or empty implies ..." sounds better than "nil implies ...".

Reply:
Done.